examples/ios/swift-2.2/RealmExamples.xcodeproj/project.pbxproj

```diff
@@ -1089,6 +1141,56 @@
 			};
 			name = Release;
 		};
+		E8F14D0F1CF8FD0C00564AF5 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				CLANG_ANALYZER_NONNULL = YES;
```

Comment:
The only ones I could find being also defined at the project level were `CLANG_ENABLE_MODULES` and `IPHONEOS_DEPLOYMENT_TARGET`. Are there others?

I'm not even sure if I'm doing this right, I'm looking at the build settings for `E80575AF19BA55E700376620` and `E80575B019BA55E700376620`.